### PR TITLE
Adding views and maps for big_maps keys on multi_asset template

### PR DESCRIPTION
**Pull request by zamrokk**
I changed for both mligo and jsligo

Beware that jsligo will not copile as %bytes literal from string does not yet exists. It never worked in fact even in previous version of jsligo files

### CODE REVIEW

**Comment by zamrokk** [comment]
I fixed the tests

**Comment by zamrokk** [comment]
I move the version to 1.0.9

File: lib/fa2/asset/multi_asset.mligo [hunk 1]
```diff
@@ -167,12 +170,19 @@ let transfer : transfer -> storage -> operation list * storage =
...
-   let process_single_transfer (ledger, t:Ledger.t * transfer_from ) =
+   let process_single_transfer (ledger, t:Ledger.t * transfer_from) =
       let {from_;txs} = t in
       let ledger     = List.fold_left (process_atomic_transfer from_) ledger txs in
       ledger
    in
    let ledger = List.fold_left process_single_transfer s.ledger t in
+   //refresh set of owner,token_ids
+   let s = List.fold_left 
+            (fun (s , tf :storage * transfer_from ) ->
```
Comment (d-plaindoux) [inline lib/fa2/asset/multi_asset.mligo:181]: Types can be removed here

Comment (zamrokk) [reply lib/fa2/asset/multi_asset.mligo:181]: if I remove typing

```
ligo compile contract ./lib/fa2/asset/multi_asset.mligo 
File "./lib/fa2/asset/multi_asset.mligo", line 184, characters 29-35:
183 |                            (fun (s , at : storage * atomic_trans) -> {s with owner_token_ids = Set.add (at.to_ , at.token_id) s.owner_token_ids })
184 |                            s tf.txs)
185 |             s t in

Invalid argument.
Expected a record, but got an argument of type "^a".
Hint: "^a" represent placeholder type(s). Adding additional annotations may resolve your error.
```

Comment (d-plaindoux) [reply lib/fa2/asset/multi_asset.mligo:181]: Okay I'll check it

**Review by d-plaindoux — APPROVED** [review]

**Pull request merged**